tests/test_edgeql_syntax.py

```diff
@@ -755,6 +755,11 @@ def test_edgeql_syntax_required_01(self):
         SELECT REQUIRED (User.groups.description);
         """
 
+    def test_edgeql_syntax_optional_01(self):
+        """
+        SELECT OPTIONAL (User.groups.description);
```

Comment:
This is actually dead syntax and needs to be removed.  We decided to express this via a cast expression.

Reply:
I was wondering about it hence I left it unimplemented. OK, will remove the test.